DependencyInjection/Compiler/AddAuditEntityCompilerPass.php

```diff
@@ -14,10 +14,6 @@
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
-/*
- *
- * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
```

Comment:
The header is juste here for the license and always refer to the same person.

The ```@author``` tag is where we get the actual author of the class.